sqld/src/http/mod.rs

```diff
@@ -211,7 +211,7 @@ async fn handle_request(
         }
     }
     match (req.method(), req.uri().path()) {
-        (&Method::POST, "/") => handle_query(req, sender).await,
+        (&Method::POST, "/queries") => handle_queries(req, sender).await,
```

Comment:
As discussed on the call. Let's try returning 301 and maybe the clients will keep working.